{
	"schema_version": "1.6.0",
	"id": "ALBA-2021:4338",
	"modified": "2021-11-12T10:20:56Z",
	"published": "2021-11-09T09:02:05Z",
	"summary": "python-coverage bug fix and enhancement update",
	"details": "For detailed information on changes in this release, see the AlmaLinux Release Notes linked from the References section.",
	"affected": [
		{
			"package": {
				"ecosystem": "AlmaLinux:8",
				"name": "platform-python-coverage"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "4.5.1-9.el8"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://github.com/AlmaLinux/osv-database/blob/master/advisories/almalinux8/ALBA-2021:4338.json"
			}
		},
		{
			"package": {
				"ecosystem": "AlmaLinux:8",
				"name": "python3-coverage"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "4.5.1-9.el8"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://github.com/AlmaLinux/osv-database/blob/master/advisories/almalinux8/ALBA-2021:4338.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://errata.almalinux.org/8/ALBA-2021-4338.html"
		}
	]
}
